WdtBase.cpp

```diff
@@ -117,9 +118,12 @@ const string WdtTransferRequest::TRANSFER_ID_PARAM{"id"};
 const string WdtTransferRequest::PROTOCOL_VERSION_PARAM{"protocol"};
 const string WdtTransferRequest::DIRECTORY_PARAM{"dir"};
 const string WdtTransferRequest::PORTS_PARAM{"ports"};
+const string WdtTransferRequest::START_PORT_PARAM{"start_port"};
+const string WdtTransferRequest::NUM_PORTS_PARAM{"num_ports"};
 
 WdtTransferRequest::WdtTransferRequest(const vector<int32_t>& ports) {
```

Comment:
const vector<int32_t> &pv
(cosmetic: & at the right place, we should really figure out why is clang format not fixing those, 2nd time this come up)
but more importantly will avoid the this-> in next 2 lines